src/rust/inetstack/protocols/mod.rs

```diff
@@ -27,7 +27,7 @@ use ::std::slice::ChunksExact;
 
 // Maximum header size of all possible headers.
 pub const MAX_HEADER_SIZE: usize =
-    tcp::MAX_TCP_HEADER_SIZE + ipv4::IPV4_HEADER_MAX_SIZE as usize + ethernet2::ETHERNET2_HEADER_SIZE;
+    tcp::MAX_TCP_HEADER_SIZE + ipv4::IPV4_HEADER_MAX_SIZE as usize + layer2::ETHERNET2_HEADER_SIZE;
```

Comment:
Encoding the unit of size in these names would be super helpful to the reader.

Reply:
I am going to make this change in another PR as well because it would change a bunch of things that are unrelated to this already large PR.